### ISSUE TITLE
MS Basic string sounds do not switch between arco and pizz

### ISSUE THREAD

**Issue by adavismusic89** [musescore/MuseScore#24427]
### Issue type

General playback bug

### Description with steps to reproduce

1. Create a score using one or more of the MS Basic string sounds like Violin, Viola, Cello, or Contrabass.
2. Using the text palette apply pizz. to one or more notes
3. You can also apply arco to one of more notes
4. Playback of the sounds remains arco for the entire part

I'm using the following:
OS: macOS 14.6, Arch.: x86_64, MuseScore Studio version (64-bit): 4.4.0-242390800, revision: github-musescore-musescore-0fcd11b

### Supporting files, videos and screenshots

[MS Basic Sound Pizz:Arco not switching.mscz.zip](https://github.com/user-attachments/files/16858683/MS.Basic.Sound.Pizz.Arco.not.switching.mscz.zip)

### What is the latest version of MuseScore Studio where this issue is present?

4.4.0-242390800, revision: github-musescore-musescore-0fcd11b

### Regression

Yes, this used to work in a previous version of MuseScore 4.x

### Operating system

macOS 14.6

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by MarcSabatella** [comment]
It seems you have set your sound explicitly to "Violin" instead of using the "Choose automatically" option.  Setting a sound explicitly defeats the automatic switching that would normally occur.  Perhaps there should be a way MuseScore could manage this anyhow, but as it is, that's what is going on here.

**Comment by RomanPudashkin** [comment]
Yep, not a regression (at least compared to 4.3.2). As @MarcSabatella correctly pointed out, you can fix pizz by clicking on the "Choose automatically" option in the mixer

**Comment by adavismusic89** [comment]
Thank you gentlemen.  I would agree with Marc that if there was a way for it to be managed, that would be great.

**Comment by MarcSabatella** [comment]
To be clear - it *is* managed automatically under normal circumstances.  It’s only because you went out your way to select a specific sound instead of using the default “choose automatically” that MuseScore cannot, in fact, choose automatically anymore like it normally would.  But sure, it would be nice to still somehow manage the pizz/arco thing automatically even when for some reason you override the normal default for the basic sound itself.
